antarest/study/storage/variantstudy/model/command/update_layer_name.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2025, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+from typing import List
+
+from typing_extensions import override
+
+from antarest.study.business.model.layer_model import LayerUpdate, update_layer_name
+from antarest.study.dao.api.study_dao import StudyDao
+from antarest.study.storage.variantstudy.model.command.common import CommandName, CommandOutput
+from antarest.study.storage.variantstudy.model.command.icommand import ICommand
+from antarest.study.storage.variantstudy.model.command_listener.command_listener import ICommandListener
+from antarest.study.storage.variantstudy.model.model import CommandDTO
+
+
+class UpdateLayerName(ICommand):
```

Comment:
I think we should use just `UpdateLayer` since later we'll add the possibility of updating also areas (I think ?)

Reply:
I agree with you, that is what I have written in the PR description. I will change it now